tools/sphere.c

```diff
@@ -37,40 +37,39 @@ extra_init() {
 
 // Apply Slooper and Sloopee shadows, respectively
 int do_sloop( string arg ) {
+    if ( !arg ) {
+        return notify_fail( "What do you want to sloop?" );
+    }
     object slooper = THISP;
-    object shad, shadtwo;
-
-    if ( !arg && !FINDP( arg ) ) {
-        ansi_tell( slooper, "You can only sloop a living being.", BOLD_PURPLE );
-        return 1;
+    if ( !IsWizard( slooper ) ) {
+        return notify_fail( "You cannot summon the power needed." );
+    }
+    object sloopee = FINDP ( arg );
+    if ( !sloopee ) {
+        return notify_fail( "You can only sloop a player." );
```

Comment:
Small thing but what about "No player found to sloop." instead? If I do "sloop asdf", your message sounds like there's an 'asdf' out there to sloop, I just can't, because it's not a player. It's definitely an edge case, but it feels like the message doesn't quite match the check that's being done.

We might also want to think about changing the check, and using Opie instead of FINDP() to search for a player. This would let us find all possible matches to their argument, and then reject them if they are not players.